scripts/eslint.sh

```diff
@@ -3,5 +3,4 @@
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
 
-. "$(dirname "$0")/_/husky.sh"
-cd frontend && npm run lint:staged
+cd frontend && yarn && npm run lint:fix
```

Comment:
Although this can work both under root dir or `frontend` dir, but still require developers to install node in their env.

Reply:
And we can also remove the `yarn` cmd here. This will make commit failed if no node_modules exist

Reply:
`lint:fix` will run eslint over the entire frontend folder on pre-commit. Thus will be slow and may effect files not being committed.
I use lint-staged to run eslint on staged files only.

Reply:
Thanks. Add it back now.